### DISCUSSION TITLE
Always have one notification, but there are nothing

### DISCUSSION THREAD

**Discussion by ShallmentMo** [community/community#119789] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="1029" alt="image" src="https://github.com/community/community/assets/1210262/2b46c170-6d87-454b-9991-f0eb8a36b4b0">

**Comment by Evgeny-ai** [comment]
Попробуй нажать кнопку Select all и нажми Done
